### PR TITLE
Fix size of dashboard links on hover

### LINKED ISSUE #270: Reduce size of dashboard links (hover)
There are links at the bottom of the dashboard. When you hover your mouse on these links, the background color changes to a grey tone. However, if you compare with the table items above, or with the separator, it is possible to notice that the dashboard links area is going a bit over and is misaligned.

image

### PR DESCRIPTION
**Pull request by ashnewport**
Remove `row` prop from `<v-layout>` to align hover with sibling elements and maximise use of space.

Closes #270

### CODE REVIEW

**Comment by kinow** [comment]
Looking perfect now!

image

@ashnewport could you please read CONTRIBUTING.md and add yourself to the contributors list, please? Once that's done this one should be ready to be reviewed. Small change, one review should be enough.

Thanks!

**Comment by ashnewport** [comment]
@kinow thanks for the review. I've read and added my name to CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ below.
  - Martin Ryan
  - Tim Pillinger
  - Alex Szabó
+ - Ash Newport
```
Comment (kinow) [inline CONTRIBUTING.md]: Awesome! Thanks!!!

**Comment by kinow** [comment]
Small change, already tested, was just waiting for CLA right updated. One review should do, merged! 

And thank you for your contribution @ashnewport 
Bruno

**Pull request merged**